core/src/main/resources/lib/layout/html.jelly

```diff
@@ -1,171 +0,0 @@
-<!--
```

Comment:
This file only had sense for the Setup Wizard and was a copy of `layout.jelly`. It was created for implementing the full screen layout. Now, different layouts can be managed through `layout.jelly`.